### PR TITLE
Mouth Item HRP Buff

### PR DESCRIPTION
**Pull request by Geevies**
## About The Pull Request

Items held in your mouth, like knives and cigarettes, will now fall to the ground when you get knocked over. As we all know due to not being able to hold anything, SS13 spacemen lose all motor function the moment they go horizontal.

Additionally, you can now light a cigarette on the floor by shooting at it with a flamethrower, extinguish it by throwing water on it. If you're holding a lit cigarette and get extinguished, your cigarette goes out.

Running into fire while holding an unlit cigarette will now light it.

You will no longer spam scream each time you walk into fire.

## Why It's Good For The Game

It helps the world feel more alive by being realistic.

## Changelog

:cl:
add: Items held in the mouth like knives or cigarettes now fall out when you get knocked over.
add: You can now light and extinguish cigarettes on the ground, as well as cigarettes held on someone's person by extinguishing them. Getting lit on fire while holding an unlit cigarette will now light it.
fix: You will no longer spam scream when you get lit on fire while already lit on fire.
/:cl:

### CODE REVIEW

**Comment by FoxyStalin** [comment]
Clenching is a common pain response, right?  This just feels like it'll be obnixous to us Cigarette Gamers

**Comment by MusselMain** [comment]
Tsk, all these cigarette/cigar nerfs, when are we gonna get a buff?

**Comment by kiVts** [comment]
Me: Puts CLF3 in cigs
Me:  THrows them on LV bridge on gap in close prooximity
Unsuspecting xeno: Hm I I wounder why this host throwing sigs around hmmm.
Me: Flames cigs.

**Comment by c4xmaniac** [comment]
Dropping the knife is big nerf to my synth drip f u

File: code/modules/projectiles/gun_attachables.dm [hunk 1]
```diff
@@ -285,6 +285,11 @@ Defined in conflicts.dm of the #defines folder.
 	hud_offset_mod = -4
 	var/pry_delay = 3 SECONDS
 
+/obj/item/attachable/bayonet/Initialize(mapload, ...)
```
Comment (spookydonut) [inline code/modules/projectiles/gun_attachables.dm:288]: Why not in /obj/item/Initialize

**Review by spookydonut — APPROVED** [review]

**Pull request merged**